docs/spec/class-compat.rst

```diff
@@ -10,7 +10,7 @@ Class type compatibility
 
 (Originally specified in :pep:`526`.)
 
-A covariant type ``ClassVar[T_co]`` exists in the ``typing``
+A covariant :term:`type qualifier` ``ClassVar[T_co]`` exists in the ``typing``
```

Comment:
I don't think it makes sense to talk about variance in the context of `ClassVar`. In most languages, this would be a keyword (like `static` in TypeScript).

Reply:
I noticed this too but left it alone because it seemed unrelated. However, we might as well change it now too.